docs/libcudacxx/extended_api/math/round_down.rst

```diff
@@ -20,10 +20,14 @@ The function computes the round down to the largest multiple of an integral or e
 
 ``value`` rounded down to the largest multiple of ``base_multiple`` less than or equal to ``value``. If ``value`` is already a multiple of ``base_multiple``, returns ``value``.
 
+**Mandates**
```

Comment:
I believe those are actually 
```suggestion
**Constraints**
```

Reply:
Correct, since we don't `static_assert` but SFINAE out.